package.json

```diff
@@ -37,11 +37,9 @@
     "@babel/register": "^7.8.6",
     "chai": "^4.2.0",
     "detect-node": "^2.0.4",
-    "eslint": "^6.8.0",
-    "eslint-config-airbnb": "^18.0.1",
-    "eslint-plugin-import": "^2.20.1",
-    "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.19.0",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.24.2",
```

Comment:
The `eslint-plugin-react` and `eslint-plugin-jsx-a11y` dependencies are no longer needed (not required by the [AirBnB base config](https://www.npmjs.com/package/eslint-config-airbnb-base)).